change/@ni-nimble-components-6bfc2c11-64d5-45f8-85b7-85d6afbf3d5f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@mollykreis The PR from a very brief scroll through is looking pretty solid. I'm likely not going to get to a full review for a few days due to NI-Tech stuff, would you be okay doing a secondary buddy review to get any feedback you could find sooner?